netbox/netbox/views/generic/bulk_views.py

```diff
@@ -160,6 +160,11 @@ def get(self, request):
         # Render the objects table
         table = self.get_table(self.queryset, request, has_bulk_actions)
 
+        filterset_form = self.filterset_form(request.GET, label_suffix='') if self.filterset_form else None
+
+        if hasattr(self, 'filterset_form'):
+            table.filterset_form = filterset_form
+
```

Comment:
```suggestion
        # Check for filterset_form on this view, if a form exists, apply to context and table, otherwise set to None
        filterset_form = None
        if hasattr(self, 'filterset_form'):
            filterset_form = self.filterset_form(request.GET, label_suffix='')
            table.filterset_form = filterset_form

```